rom/dev/src/fht.rs

```diff
@@ -151,10 +151,7 @@ impl FhtDataStore {
 pub fn initialize_fht(env: &mut RomEnv) {
     let pdata = &env.persistent_data.get();
 
-    cprintln!(
-        "[fht] Storing FHT @ 0x{:08X}",
-        &pdata.fht as *const _ as usize
-    );
+    cprintln!("[fht] FHT @ 0x{:08X}", &pdata.fht as *const _ as usize);
```

Comment:
nit: Should we do a separate PR for cleaning up the log strings?

Another option would be to squash the actual feature into one commit and then split the log changes into a separate commit.